examples/with-gatsby/packages/ui/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  root: true,
+  extends: ["custom/react"],
+  rules: {
+    "@typescript-eslint/no-unused-vars": "off",
```

Comment:
Required because tsup needs react to be in imported, but it's never technically uses

Reply:
> tsup needs react to be in imported,

wut